From bae0e5cdd8d02aeea0e6c28f17d1b21a32bd6f63 Mon Sep 17 00:00:00 2001
From: Tejas Patel <tejas.patel@xilinx.com>
Date: Tue, 18 Feb 2020 02:15:16 -0800
Subject: [PATCH 1230/1851] drivers: soc: xilinx: fix build failure due to
 missing COMMON_CLK dependency

commit 3b22c9d611d891b8ac96455fbbe2b653671d2ba9 from
https://github.com/Xilinx/linux-xlnx.git

This patch fix the build failure on m68k.

xlnx_vcu_clk is clock provider which depends on COMMON_CLK.
Compilation breaks if COMMON_CLK is not enabled, as xlnx_vcu_clk
is using common clock framework callbacks, api and structures.

Reported-by: kbuild test robot <lkp@intel.com>
Signed-off-by: Tejas Patel <tejas.patel@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/soc/xilinx/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/xilinx/Kconfig b/drivers/soc/xilinx/Kconfig
index 4cce9f315f4e..9767db279ba4 100644
--- a/drivers/soc/xilinx/Kconfig
+++ b/drivers/soc/xilinx/Kconfig
@@ -4,7 +4,7 @@ menu "Xilinx SoC drivers"
 config XILINX_VCU
 	tristate "Xilinx VCU logicoreIP Init"
 	select MFD_CORE
-	depends on HAS_IOMEM
+	depends on HAS_IOMEM && COMMON_CLK
 	help
 	  Provides the driver to enable and disable the isolation between the
 	  processing system and programmable logic part by using the logicoreIP
-- 
2.31.1

